package/network/utils/iproute2/Makefile

```diff
@@ -90,11 +98,12 @@ TARGET_CFLAGS += -DHAVE_SETNS
 TARGET_CFLAGS += -ffunction-sections -fdata-sections
```

Comment:
please add IP_CONFIG_SETNS to Concifg file instead of manually activating HAVE_SETNS here. Normally the Config file gets generated with such information.